Demo/Demo/ViewModels/PayPalWebViewModel.swift

```diff
@@ -70,64 +72,54 @@ class PayPalWebViewModel: ObservableObject, PayPalWebCheckoutDelegate {
         }
     }
 
-    func paypalWebCheckoutSuccessResult(checkoutResult: PayPalWebState.CheckoutResult) {
-        DispatchQueue.main.async {
-            self.state.checkoutResultResponse = .loaded(checkoutResult)
-        }
-    }
-
-    func paypalWebCheckoutFailureResult(checkoutError: CorePayments.CoreSDKError) {
-        DispatchQueue.main.async {
-            self.state.checkoutResultResponse = .error(message: checkoutError.localizedDescription)
-        }
-    }
```

Comment:
These methods are only doing 1 thing so moved this logic into the delegates directly vs calling a method